sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1288,3 +1288,31 @@ def analyzer(doc):
 
     with pytest.raises(Exception, match="testing"):
         Estimator(analyzer=analyzer, input='file').fit_transform([f])
+
+
+@pytest.mark.parametrize(
+    'Vectorizer', [CountVectorizer]
```

Comment:
If TfidfVectorizer and HashingVectorizer also pass this test (as they inherit from CountVectorizer) please add them here.

Reply:
Actually , I did try all three  earlier but  it failed .  The original post also mentioned
from sklearn.feature_extraction.text import **CountVectorizer**
so I stuck to that  
<img width="748" alt="Screen Shot 2019-08-10 at 10 26 05 AM" src="https://user-images.githubusercontent.com/4666017/62822967-483dbd80-bb59-11e9-8cd3-2e9296ab6bf9.png">

Reply:
Sure . Will try and get back to you .